.travis.yml

```diff
@@ -77,7 +77,7 @@ script:
     fi
   - |
     # Build presto-server-rpm for later artifact upload
```

Comment:
I think we can delete this whole section now, since `MAVEN_CHECKS` does an install of all modules (including the server RPM)